RELEASE.md

```diff
@@ -16,6 +16,7 @@ Please follow the established format:
 ## Bug fixes and other changes
 
 <!-- Add release notes for the upcoming release here -->
+* Added information for multiple pipelines in `nodes.json` (#192)
```

Comment:
I don't think it's really worth changing, but I wanted to note that the rest of this document uses hyphens for bullets, not asterixes. Also this document typically uses present tense, not past tense, as this is recommended commit title best practice.

Reply:
@richardwestenra Thanks! I can change it. Very confusingly, Kedro release note is past tense + bullet points.... :(